src/vmm/src/devices/virtio/net/device.rs

```diff
@@ -1323,6 +1334,28 @@ pub mod tests {
         assert!(!tap_traffic_simulator.pop_rx_packet(&mut []));
     }
 
+    #[test]
+    fn test_tx_big_frame() {
+        let mut th = TestHelper::get_default();
+        th.activate_net();
+        let tap_traffic_simulator = TapTrafficSimulator::new(if_index(&th.net().tap));
+
+        // Send an invalid frame (too big, maximum buffer is MAX_BUFFER_SIZE).
+        th.add_desc_chain(NetQueue::Tx, 0, &[(0, 100000, 0)]);
```

Comment:
nit: can we use a number relative to `MAX_BUFFER_SIZE` (such as 1.5x) instead of a literal, to be explicit?

Reply:
Did `MAX_BUFFER_SIZE + 1`